tools/deployment/make_linux_package.sh

```diff
@@ -37,6 +37,8 @@ SYSTEMD_SYSCONFIG_DST="/etc/sysconfig/osqueryd"
 CTL_SRC="$SCRIPT_DIR/osqueryctl"
 PACKS_SRC="$SOURCE_DIR/packs"
 PACKS_DST="/usr/share/osquery/packs/"
+OSQUERY_POSTINSTALL=$OSQUERY_POSTINSTALL
```

Comment:
We can assign them "" but I will have to check after to see if the env vars are set to overwrite them if you want to build with ````make package````.

i.e.:
````
OSQUERY_CONFIG_SRC=/some/directory/osquery.example.conf OSQUERY_POSTINSTALL=./postinst.sh make package
````
If you are ok with that then I can change it.